### PR TITLE
Prettify everything that isn't 💄💯

### PR DESCRIPTION
**Pull request by sndrs**
_depends on #9424 and #9446..._

## What does this change?

- conforms all files the prettier config
- re-enables CI prettier checks for the repo (now that everything is formatted they should pass again)
- disables the `max-len` lint rule for app-rendering’s `parser.ts` (will need a refactor to address)

## Why?

Follow up to #9424, bringing everything into line. In _theory_, this should be the last ever commit of pure formatting changes (it won't be, obviously)

## Reviewing

This is def one for hiding whitespace changes https://github.com/guardian/dotcom-rendering/pull/9425/files?w=1.

The main changes should be formatting. For the formatting changes, the only think I did was run this at the root 

```sh
yarn prettier:write
```

### Before

```sh
[warn] .github/dependabot.yml
[warn] .github/ISSUE_TEMPLATE/request-for-change.md
[warn] .github/workflows/amp.yml
[warn] .github/workflows/ar-nodejs.yml
[warn] .github/workflows/ar-pr-deployment.yml
[warn] .github/workflows/cicd.yml
[warn] .github/workflows/cypress.yml
[warn] .github/workflows/delete-old-packages.yml
[warn] .github/workflows/lint.yml
[warn] .github/workflows/permissions-advisor.yml
[warn] .github/workflows/publish.yml
[warn] .github/workflows/snyk.yml
[warn] .github/workflows/stale.yml
[warn] .vscode/dcr.code-snippets
[warn] .vscode/extensions.json
[warn] apps-rendering/.storybook/main.js
[warn] apps-rendering/artifact.json
[warn] apps-rendering/cdk.json
[warn] apps-rendering/config/jest.config.js
[warn] apps-rendering/config/jestSetup.js
[warn] apps-rendering/config/rendered-items-assets-template.html
[warn] apps-rendering/config/tsconfig.cdk.json
[warn] apps-rendering/config/tsconfig.client.json
[warn] apps-rendering/config/tsconfig.server.json
[warn] apps-rendering/config/tsconfig.test.json
[warn] apps-rendering/pull_request_template.md
[warn] apps-rendering/README.md
[warn] apps-rendering/riff-raff.yaml
[warn] apps-rendering/src/articleFormat.test.ts
[warn] apps-rendering/src/atoms.ts
[warn] apps-rendering/src/bodyElement.ts
[warn] apps-rendering/src/campaign.test.ts
[warn] apps-rendering/src/components/Callout/Callout.stories.tsx
[warn] apps-rendering/src/components/Callout/callout.test.ts
[warn] apps-rendering/src/components/Callout/calloutBlock.tsx
[warn] apps-rendering/src/components/Callout/calloutComponents.tsx
[warn] apps-rendering/src/components/Callout/index.tsx
[warn] apps-rendering/src/components/Callout/styles.ts
[warn] apps-rendering/src/components/Layout/Layout.stories.tsx
[warn] apps-rendering/src/components/Paragraph/Paragraph.stories.tsx
[warn] apps-rendering/src/components/PinnedPost/PinnedPost.stories.tsx
[warn] apps-rendering/src/components/RichLink/RichLink.stories.tsx
[warn] apps-rendering/src/components/SpecialReportAltAtom/index.tsx
[warn] apps-rendering/src/components/SpecialReportAltAtom/SpecialReportAltAtom.stories.tsx
[warn] apps-rendering/src/contributer.test.ts
[warn] apps-rendering/src/date.test.ts
[warn] apps-rendering/src/image/srcsets.test.ts
[warn] apps-rendering/src/item.test.ts
[warn] apps-rendering/src/newsletter.test.ts
[warn] apps-rendering/src/palette/background.ts
[warn] apps-rendering/src/palette/text.ts
[warn] apps-rendering/src/parser.md
[warn] apps-rendering/src/parser.ts
[warn] apps-rendering/src/result.test.ts
[warn] apps-rendering/src/span.test.ts
[warn] apps-rendering/src/styles.test.ts
[warn] apps-rendering/tsconfig.json
[warn] CONTRIBUTING.md
[warn] dotcom-rendering/.storybook/decorators/configContextDecorator.tsx
[warn] dotcom-rendering/.stylelintrc
[warn] dotcom-rendering/cdk.json
[warn] dotcom-rendering/cdk/lib/dotcom-rendering.ts
[warn] dotcom-rendering/cypress/fixtures/richLink.json
[warn] dotcom-rendering/cypress/fixtures/shareCount.json
[warn] dotcom-rendering/docs/ads/apps-ads.md
[warn] dotcom-rendering/docs/ads/web-ads.md
[warn] dotcom-rendering/docs/architecture/000-architecture-decision-records.md
[warn] dotcom-rendering/docs/architecture/002-componentisation.md
[warn] dotcom-rendering/docs/architecture/003-react.md
[warn] dotcom-rendering/docs/architecture/006-support-for-frontend-only.md
[warn] dotcom-rendering/docs/architecture/007-css-in-js.md
[warn] dotcom-rendering/docs/architecture/008-static-types.md
[warn] dotcom-rendering/docs/architecture/009-storybook.md
[warn] dotcom-rendering/docs/architecture/011-amp.md
[warn] dotcom-rendering/docs/architecture/014-client-side-computation.md
[warn] dotcom-rendering/docs/architecture/018-react-hooks.md
[warn] dotcom-rendering/docs/architecture/023-script-loading.md
[warn] dotcom-rendering/docs/architecture/027-pictures.md
[warn] dotcom-rendering/docs/architecture/3rd-party-technical-review/001-modi-poc.md
[warn] dotcom-rendering/docs/architecture/3rd-party-technical-review/003-gracenote.md
[warn] dotcom-rendering/docs/architecture/amp/000-structure-for-initial-milestone.md
[warn] dotcom-rendering/docs/architecture/historic-adrs/dotenv.md
[warn] dotcom-rendering/docs/commercial/liveblog-server-side-ad-slots.md
[warn] dotcom-rendering/docs/contracts/001-commercial-selectors.md
[warn] dotcom-rendering/docs/contributing/code-style.md
[warn] dotcom-rendering/docs/contributing/detailed-setup-guide.md
[warn] dotcom-rendering/docs/contributing/how-to.md
[warn] dotcom-rendering/docs/contributing/README.md
[warn] dotcom-rendering/docs/development/ab-testing-in-dcr.md
[warn] dotcom-rendering/docs/development/detecting-and-correcting-vulnerabilities-with-snyk.md
[warn] dotcom-rendering/docs/development/run-prod-bundle-locally.md
[warn] dotcom-rendering/docs/development/track-user-behaviour.md
[warn] dotcom-rendering/docs/duplication.md
[warn] dotcom-rendering/docs/elements/Embed.md
[warn] dotcom-rendering/docs/elements/Image.md
[warn] dotcom-rendering/docs/elements/Richlink.md
[warn] dotcom-rendering/docs/elements/Text.md
[warn] dotcom-rendering/docs/email-newsletters-page.md
[warn] dotcom-rendering/docs/principles/browser-support.md
[warn] dotcom-rendering/docs/principles/performance-budgets.md
[warn] dotcom-rendering/docs/resources.md
[warn] dotcom-rendering/docs/snyk/how-to.md
[warn] dotcom-rendering/docs/testing.md
[warn] dotcom-rendering/docs/tracking/001-sentry.md
[warn] dotcom-rendering/docs/youtube/youtube-variations-2021-11-17.md
[warn] dotcom-rendering/fixtures/manual/qandaAtom.tsx
[warn] dotcom-rendering/README.md
[warn] dotcom-rendering/scripts/deploy/riff-raff.yaml
[warn] dotcom-rendering/scripts/gen-stories/get-stories.mjs
[warn] dotcom-rendering/scripts/islands/islands.html
[warn] dotcom-rendering/scripts/perf/k6/article-nier-automata.json
[warn] dotcom-rendering/src/client/debug/README.md
[warn] dotcom-rendering/src/components/SignInGate/README.md
[warn] dotcom-rendering/src/model/article-schema.json
[warn] dotcom-rendering/src/model/block-schema.json
[warn] dotcom-rendering/src/model/front-schema.json
[warn] dotcom-rendering/src/model/newsletter-page-schema.json
[warn] dotcom-rendering/src/model/tag-front-schema.json
[warn] dotcom-rendering/src/server/dev-index.html
[warn] dotcom-rendering/src/static/css/print.css
[warn] dotcom-rendering/src/static/js/curl-with-js-and-domReady.js
[warn] dotcom-rendering/stories/generated/Card.stories.tsx
[warn] dotcom-rendering/stories/generated/CardReadme.stories.jsx
[warn] dotcom-rendering/stories/generated/Layout.stories.tsx
[warn] dotcom-rendering/stories/generated/LayoutReadme.stories.jsx
[warn] dotcom-rendering/tsconfig.build.json
[warn] dotcom-rendering/tsconfig.cdk.json
[warn] dotcom-rendering/tsconfig.json
[warn] dotcom-rendering/tsconfig.test.json
[warn] dotcom-rendering/window.guardian.ts
```
### After
👄

### CODE REVIEW

**Review by abeddow91 — APPROVED** [review]
🧹  thank you @sndrs

**Review by mxdvl — APPROVED** [review]
Is this commit worth adding to [`.git-blame-ignore-revs`](https://github.com/guardian/dotcom-rendering/blob/3a3333d41f6e42ebf5e416c93535e7e7b7101131/.git-blame-ignore-revs)?

File: apps-rendering/src/parser.ts [hunk 1]
```diff
@@ -536,13 +536,15 @@ const map7 =
 				),
 		);
 
+/* eslint-disable max-len -- this is a lot of nesting, can it be refactored? */
```
Comment (sndrs) [inline apps-rendering/src/parser.ts:539]: just fyi @JamieB-gu

**Review by mxdvl — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -15143,7 +15143,7 @@ prebid.js@guardian/prebid.js#0b4cccd:
     core-js "^3.13.0"
     core-js-pure "^3.13.0"
     criteo-direct-rsa-validate "^1.1.0"
-    crypto-js "^3.3.0"
+    crypto-js "^4.2.0"
```
Comment (mxdvl) [inline yarn.lock:15146]: 👏

**Pull request merged**